scripts/install/mylar.sh

```diff
@@ -0,0 +1,100 @@
+. /etc/swizzin/sources/functions/utils
+app_name="mylar"
+if [ -z "$MYLAR_OWNER" ]; then
+    if ! MYLAR_OWNER="$(swizdb get "$app_name/owner")"; then
+        MYLAR_OWNER="$(_get_master_username)"
+        echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+        swizdb set "$app_name/owner" "$MYLAR_OWNER"
+    fi
+else
+    echo_info "Setting ${app_name^} owner = $MYLAR_OWNER"
+    swizdb set "$app_name/owner" "$MYLAR_OWNER"
+fi
+user="$MYLAR_OWNER"
+swiz_configdir="/home/$user/.config"
+app_configdir="$swiz_configdir/${app_name^}"
+app_configfile="$app_configdir/config.ini"
+app_group="$user"
+app_port="$(port 7000 11000)"
+app_servicefile="$app_name.service"
+app_dir="/opt/${app_name^}"
+app_binary="${app_name^}"
+#Remove any dashes in appname per FS
+app_lockname="${app_name//-/}"
+echo_info "Setting ${app_name^} port = ${app_port}"
+swizdb set "mylar/port" "${app_port}"
+
+if [ ! -d "$swiz_configdir" ]; then
+    mkdir -p "$swiz_configdir"
+fi
+function install_mylar() {
+    chown "$user":"$app_group" "$swiz_configdir"
+    # deps
+    apt_install python3 git
+    mkdir -p /opt/.venv/${app_name}
+    # pyenv
+    pyenv_install
+    pyenv_install_version 3.8.1
+    pyenv_create_venv 3.8.1 /opt/.venv/${app_name}/
+    # mylar
+    echo_progress_start "Grabbing the latest Mylar."
+    git clone https://github.com/mylar3/mylar3.git $app_dir >> $log 2>&1
+    echo_progress_done "Mylar downloaded."
+    echo_progress_start "Upgrading pip and installing dependencies."
+    /opt/.venv/${app_name}/bin/pip install --upgrade pip >> $log 2>&1
+    /opt/.venv/${app_name}/bin/pip install --upgrade pip >> $log 2>&1
+    /opt/.venv/${app_name}/bin/pip3 install -r $app_dir/requirements.txt >> $log 2>&1
+    echo_progress_done "Upgrading pip and installing dependencies."
+    # ownership
+    echo_progress_start "Owning the files to $user"
+    chown $user:$app_group -R $app_dir
+    chown $user:$app_group -R /opt/.venv/${app_name}
+    echo_progress_done "$user owns files."
+    echo_progress_start "Writing service file."
+    cat > /etc/systemd/system/$app_servicefile << MLR
+[Unit]
+Description=mylarr3 service
+After=nginx.service
+
+[Service]
+Type=simple
+User=$user
+
+ExecStart=/opt/.venv/${app_name}/bin/python $app_binary.py --datadir $app_configdir
+WorkingDirectory=$app_dir
+Restart=on-failure
+TimeoutStopSec=300
+
+[Install]
+WantedBy=multi-user.target
+MLR
+    if [[ -f /install/.nginx.lock ]]; then
+        interface=127.0.0.1
+    else
+        interface=0.0.0.0
+    fi
+    mkdir -p "${app_configdir}"
+    cat > "${app_configfile}" << CFG
+[Interface]
+http_port = ${app_port}
+http_host = ${interface}
+http_root = /${app_name}
+CFG
+    chown $user:$app_group -R ${app_configdir}
+    echo_progress_done "Service file written."
+    systemctl enable -q --now ${app_servicefile} || echo_warn "Mylar failed to start."
+}
+#shellcheck source=scripts/nginx/mylar.sh
+. /etc/swizzin/scripts/nginx/mylar.sh
```

Comment:
This is not standard format for this, the nginx check happens outside the nginx script. Check any other installer.